.github/workflows/build-on-commit.yml

```diff
@@ -0,0 +1,98 @@
+name: Build Commit
+on: [push]
+jobs:
+  build-elmu:
+    runs-on: ubuntu-latest
+    env:
+      NODE_ENV: development
+      ELMU_ENV: test
+    services:
+      minio:
+        image: bitnami/minio:2020.12.18
+        ports:
+          - 9000:9000
+        env:
+          MINIO_ACCESS_KEY: "UVDXF41PYEAX0PXD8826"
+          MINIO_SECRET_KEY: "SXtajmM3uahrQ1ALECh3Z3iKT76s2s5GBJlbQMZx"
+          MINIO_BROWSER: "on"
+          MINIO_DOMAIN: "localhost"
+      mongo:
+        image: mongo:4.2.11-bionic
+        ports:
+          - 27017:27017
+      maildev:
+        image: maildev/maildev:1.1.0
+        ports:
+          - 8000:8000
+    steps:
+      - name: Change user
+        run: sudo chown -R $USER:$USER ${{ github.workspace }}
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - uses: actions/cache@v2
```

Comment:
We can do this a lot simpler:
Caching for npm/yarn is automatically built-in into `actions/setup-node@2` by specifying:

~~~
- uses: actions/setup-node@v2
  with:
    node-version: '16.10.0'
    cache: 'yarn'
~~~

Then we can delete the whole cache action afterwards.
Also, note the more specific node version.

See also: https://github.com/actions/setup-node#caching-packages-dependencies